src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    public static int inputCount = 0;
+    public static Task[] tasks = new Task[100];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String input;
+        Scanner in = new Scanner(System.in);
+        int taskNumber = 0;
+        String statusOfTask;
+        String[] inputSplitter;
+
+
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?\n");
+        input = in.nextLine();
+
+
+        while (!(input.equals("bye"))) {
+
+            if (input.equals("list")) {
+                input = printList(in);
+
+            } else if (input.contains("done")) {
+                input = testTaskDone(input, in);
+
+            } else if (input.contains("todo")) {
+                Task whatToDo = toDoMethod(input);
+                System.out.println("    Got it. I've added this task:\n" + "    " + whatToDo + "\n"
+                        + "    Now you have " + inputCount + " tasks in the list.");
+
+                input = in.nextLine();
+
+            } else if (input.contains("deadline")) {
+                Task whatDeadline = deadlineMethod(input);
+                System.out.println("    Got it. I've added this task:\n" + "    " + whatDeadline + "\n"
+                        + "    Now you have " + inputCount + " tasks in the list.");
+
+                input = in.nextLine();
+
+            } else if (input.contains("event")) {
+                Task whatEvent = eventMethod(input);
+                System.out.println("    Got it. I've added this task:\n" + "    " + whatEvent + "\n"
+                        + "    Now you have " + inputCount + " tasks in the list.");
+
+                input = in.nextLine();
+
```

Comment:
Not sure if it is intended but there is some unnecessary whitespace before closing braces of if-else statements